tests/e2e/matmul/BUILD.bazel

```diff
@@ -31,6 +31,7 @@ py_binary(
         "--iree-opt-data-tiling=false",
         "--iree-llvmcpu-enable-ukernels=none",
         "--iree-llvmcpu-enable-scalable-vectorization",
+        "--iree-llvmcpu-target-triple=aarch64-unknown-unknown",
```

Comment:
Is enabling scalable vectorization and only compiling for aarch64 intended in this `e2e_matmul_cpu_nondt_` test suite, or should this suite keep default flags and those go in another test suite?

Reply:
Scalable vector is only available for aarch64 with some specific cpu features (e.g., sme, etc). Actually this test suite is running nothing, so I perhaps can remove the change. What I'm looking for was passing correct target-triple based on the target arch. E.g., `x86_64-unknown-unknown` for x86 targets and `aarch64-unknown-unknown` for aarch64 targets. It is tricky in current test suite setup, so I ended with this. The point is that `iree-llvmcpu-enable-scalable-vectorization` is reasonable only when we target aarch64 and have the CPU features.

The below one is needed because we'd query the arch to decide tile sizes in CPU codegen, but not only look at features.